blocks/header/header.js

```diff
@@ -87,11 +87,29 @@ const TEMPLATE = /* html */`
 
   <!-- Search -->
   <div class="pan-search-panel">
-      <div class="search-panel-container container">
-          <button class="search-panel-close locale-search-panel-close">
-              <i class="ion-ios-close-outline"></i>
-          </button>
+    <div class="search-panel-container container">
+      <!-- Start: Coveo Search Box Implementation -->
+      <script class="coveo-script" src="https://static.cloud.coveo.com/searchui/v2.10082/js/CoveoJsSearch.min.js"></script>
```

Comment:
could you import the js & css on `mouseover` of the search button? we don't need it on page load and it'd be better to postpone for performance